frontend/app-development/layout/PageHeader/SubHeader/SettingsModalButton/SettingsModal/SettingsModal.tsx

```diff
@@ -11,6 +11,7 @@ import { AccessControlTab } from './components/Tabs/AccessControlTab';
 import { SetupTab } from './components/Tabs/SetupTab';
 import { type SettingsModalHandle } from '../../../../../types/SettingsModalHandle';
 import { useSettingsModalMenuTabConfigs } from './hooks/useSettingsModalMenuTabConfigs';
+import { Maskinporten } from './components/Tabs/Maskinporten/Maskinporten';
```

Comment:
Should we add an `index.ts` file in the folder to export Maskinporten so that we do not get the word duplicated on imports? 😄 
```tsx
export { Maskinporten } from './Maskinporten'
```